conans/client/conf/config_installer.py

```diff
@@ -273,10 +278,14 @@ def is_config_install_scheduled(api):
     config_install_file = cache.config_install_file
     if interval is not None:
         if os.path.exists(config_install_file):
-            timestamp = os.path.getmtime(config_install_file)
-            sched = datetime.fromtimestamp(timestamp, tz=gettz())
-            sched += interval
-            now = datetime.now(gettz())
-            return now > sched
+            if _load_configs(config_install_file):
+                timestamp = os.path.getmtime(config_install_file)
+                sched = datetime.fromtimestamp(timestamp, tz=gettz())
+                sched += interval
+                now = datetime.now(gettz())
+                return now > sched
+            else:
+                api.out.warn("Skipping scheduled config install, "
```

Comment:
I would say that if it is empty, it would keep warning this all the time, at every invocation?

Reply:
I've reformulated the idea. It will warn the user, but only when scheduled, not for every new command.